packages/application-shell/src/components/quick-access/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import Loadable from 'react-loadable';
-import ButlerContainer from './butler-container';
+// import Loadable from 'react-loadable';
+// import ButlerContainer from './butler-container';
 import * as gtm from '../../utils/gtm';
 import trackingEvents from './tracking-events';
-
-const QuickAccessModal = Loadable({
-  loader: () => import('./quick-access' /* webpackChunkName: "quick-access" */),
-  loading: ButlerContainer,
-});
+import QuickAccessModal from './quick-access';
+// TODO: make this work in different environments.
```

Comment:
one way to deal with this would be to set an env variable, and change the path based on that.

the current issue is that `./quick-access` doesn't exist in the context of `dist`